src/components/SpecialResolution/CreateSpecialResolution.vue

```diff
@@ -300,23 +297,28 @@ export default class CreateSpecialResolution extends Mixins(DateMixin) {
    * True if date is >= the minimum (ie, today) and <= the maximum (ie, the 10th day).
    * This is used for Vue form validation (in Date Rules above).
    */
-  isBetweenDates (minDate: string, maxDate: string, dateStrToValidate: string): boolean {
+  private isValidDateRange (minDate: Date, maxDate: Date, dateStrToValidate: string): boolean {
     if (!dateStrToValidate) { return true }
-    return (new Date(dateStrToValidate) >= new Date(minDate)) && (new Date(dateStrToValidate) <= new Date(maxDate))
+    // Input is in the format of MM dd, yyyy - only compare year/month/day (ignore time)
+    const utcDateStr = new Date(dateStrToValidate + ' 00:00 UTC').toISOString().split('T')[0]
```

Comment:
I'll revisit this with a comparison to `createUtcDate` for the File and Pay ticket: 

https://github.com/bcgov/entity/issues/13231

There's some backend changes that need to be done in here:
https://github.com/bcgov/entity/issues/13459

To provide the most recent filing date

Reply:
FYI,

"Complex dates" (eg, yyyy-mm-ddThh:mm:ss.xxx+00:00) in the BE are UTC.

"Simple dates" (eg, yyyy-mm-dd) should be Pacific dates, but some might not be.